src/status_im/transport/handlers.cljs

```diff
@@ -155,3 +160,11 @@
      (if message
        (handlers-macro/merge-fx cofx fx (message/receive message chat-id signature))
        fx))))
+
+(re-frame/reg-fx
+ :confirm-message-processed
+ (fn [messages]
+   (doseq [{:keys [web3 js-obj]} messages]
+     (.. web3
+         -shh
+         (confirmMessagesProcessed #js [js-obj] (fn [_ _]))))))
```

Comment:
Can we map and makes a single call?

Or we need to use a different web3 object for each message?

Reply:
oh sure, i forgot about this

Reply:
done